can/common.cc

```diff
@@ -108,7 +108,7 @@ void init_crc_lookup_tables() {
   gen_crc_lookup_table_16(0x1021, crc16_lut_xmodem);    // CRC-16 XMODEM for HKG CAN FD
 }
 
-unsigned int volkswagen_mqb_checksum(uint32_t address, const Signal &sig, const std::vector<uint8_t> &d) {
+unsigned int volkswagen_mxb_checksum(uint32_t address, const Signal &sig, const std::vector<uint8_t> &d) {
```

Comment:
what do you think of mxb -> mqb_mlb for better grep-ability?

Reply:
I'm okay with it. The whole set of abstractions needs another look, was focused on passing tests tonight, will look at it with fresh eyes later.